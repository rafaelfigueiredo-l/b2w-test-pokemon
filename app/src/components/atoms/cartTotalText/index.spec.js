import CartTotalText from "./";

describe("<CartTotalText />", () => {
  const component = <CartTotalText>Texto</CartTotalText>;

  const mounted = mount(component);
  const wrapper = shallow(component);

  it("should mount correctly CartTotalText component", () => {
    expect(wrapper.find(".cartTotalText")).toHaveLength(1);
    expect(mounted).toMatchSnapshot();
  });

  it("should unmount correctly CartTotalText component", () => {
    expect(mounted.unmount()).toEqual({});
  });

  it("should have a children", () => {
    expect(wrapper.find(".cartTotalText").children()).toHaveLength(1);
  });
});
