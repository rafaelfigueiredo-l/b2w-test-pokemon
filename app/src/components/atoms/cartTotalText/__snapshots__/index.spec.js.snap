// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartTotalText /> should mount correctly CartTotalText component 1`] = `
<styled.span
  className="cartTotalText"
>
  <span
    className="sc-AxjAm jtICzP cartTotalText"
  >
    Texto
  </span>
</styled.span>
`;
